keylime/da/attest.py

```diff
@@ -0,0 +1,154 @@
+#!/usr/bin/python3
+import argparse
+from datetime import datetime
+
+from keylime import cloud_verifier_common, config, keylime_logging
+from keylime.da import record
+
+logger = keylime_logging.init_logging("durable_attestation_fetch_and_replay")
+
+
+def main() -> None:
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        "-u",
+        "--uuid",
+        action="store",
+        dest="agent_uuid",
+        default="all",
+        help='UUID for the agent to attest (default "all")',
+    )
+    parser.add_argument(
+        "-s",
+        "--start",
+        action="store",
+        dest="start_date",
+        default="first",
+        help='start date for attestation window in either IS0 8601 format (e.g., "2008-09-03T20:56:35"), or keyworkd "first"/"none"',
+    )
+    parser.add_argument(
+        "-e",
+        "--end",
+        action="store",
+        dest="end_date",
+        default="last",
+        help='end date for attestation window in IS0 8601 format (e.g., "2008-09-03T21:56:35"), or keyworkd "last"',
+    )
+
+    args = parser.parse_args()
```

Comment:
I think there should be a help screen that states that the systems-of-interests' durable attestation logs must not contain any IMA log entries since this would cause the attestation replay to most likely fail.

Reply:
I will add a message stating this feature is **experimental** as I did with all the rest. This should be enough to cover all the (hopefully) temporary corner cases.

Reply:
> he (hopefully) temporary  corner cases.

I guess this a matter of 'perception'.
I would just write what is known to not work.

Reply:
Ah, I see that the message about being an experimental feature is already there.